src/controllers/token/erc20/_.validation.ts

```diff
@@ -0,0 +1,13 @@
+import { body, param } from 'express-validator';
+
+const validation = {
+    getERC20Token: [param('id').exists().isString()],
```

Comment:
@GarfDev I recently discovered (not tested) express-validator is able to test for Mongo's ObjectId with the `.isMongoId()` call. Let's start using that for validation purposes!